modules/conversion-center/assets/js/frontend/handlers/contact-buttons.js

```diff
@@ -141,24 +155,36 @@ export default class ContactButtonsHandler extends Base {
 		this.elements.messageBubbleTime.innerHTML = time;
 	}
 
+	removeChatButtonAnimationClasses() {
+		const { chatButtonAnimation, visible } = this.getSettings( 'constants' );
+
+		this.elements.chatButton.classList.remove( chatButtonAnimation );
+		this.elements.chatButton.classList.add( visible );
+	}
```

Comment:
@mserino Not sure I understand. Why do you need to remove this classes? I don't see that we've done it anywhere before, so just want to make sure.

Reply:
@louiswol94 I have actually mirrored this code from the `off-canvas` widget in the pro repo. There was actually a small bug that prevented the correct functioning of the animation without removing the classes.

https://github.com/elementor/elementor-pro/blob/main/modules/off-canvas/assets/js/frontend/handlers/off-canvas.js#L161